### PR TITLE
setup configuration for database connection

### PR DESCRIPTION
**Pull request by johndy2742**
### Changes Made

- Added the ability to load environment variables from the .env file in the config.go file of the "app" package.
- Modified the config.go file to read environment variables using the godotenv package and set them in the Viper configuration.
- Updated the LoadConfig function to load and provide access to environment variables, including DBSourceMaster, DBSourceSlave, and DBDriver.

### Purpose

The changes in this PR enable the application to load environment variables from the .env file, making it easier to manage database configuration variables. This improves the flexibility and ease of configuring the application's database connections.

### Related Links

https://chat.digitalservice.id/boards/team/nii5pbyw1idwxrxjd8bp56mtkw/bte55h31wfjgp9xgry174ur3mae/vbyyag4aadi8ejgca6gtqzs55gc

Project: Event Management System
Participants: @johndy2742, @iqbal167, @alqinsidev

### CODE REVIEW

File: service/.env.example
Comment (alqinsidev) [inline service/.env.example:1]: you don't need to push the .env.example except it contain new variable

**Review by alqinsidev — APPROVED** [review]

File: service/src/app/config.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+// Config holds the application configuration.
+type Config struct {
+	DBSourceMaster string
+	DBSourceSlave  string
+	DBDriver       string
+}
+
+// LoadConfig initializes and loads the application configuration into the Config struct.
+func LoadConfig() (Config, error) {
```
Comment (iqbal167) [inline service/src/app/config.go:16]: what if the `Config` return is set with a pointer?

File: service/src/services/main.go [hunk 2]
```diff
@@ -20,11 +20,16 @@ import (
 
 func main() {
 	var (
-		app    = app.Init()
-		router = app.GetHttpRouter()
-		module = modules.Init(app)
+		app, initErr = app.Init()
```
Comment (iqbal167) [inline service/src/services/main.go:23]: you can remove the `var` keyword too, use short declaration operator

File: service/src/app/config.go (hunk 1 above)
Comment (johndy2742) [reply service/src/app/config.go:16]: it can be done like that too kang @iqbal167 , which one is preferred in jds standard?

Comment (iqbal167) [reply service/src/app/config.go:16]: It's fine to use it, but I'd like to discuss why you should use it

**Review by iqbal167 — APPROVED** [review]

**Pull request merged**